protocol/minimum-base-fee.md

```diff
@@ -64,7 +64,7 @@ PayloadAttributesV3: {
     transactions: array of DATA
     noTxPool: bool
     gasLimit: QUANTITY or null
-    eip1559Params: DATA (9 bytes) or null
+    eip1559Params: DATA (8 bytes) or null
```

Comment:
We're missing `minBaseFeeLog2` in the payload attributes, which also means we're probably not handling it properly in `ForkChoiceUpdated`. That means op-node can't instruct op-geth to build a payload with the current value of the setting.